docs/anyedge/anyedge.md

```diff
@@ -108,7 +108,7 @@ If a packet fails to validate, you can choose how to handle it. You can handle t
 + No "To" header.
  No "Via" header.
 + Request URI parse error.
-+ Bad hostname in "R-URI".
++ Bad hostname in "R-URI."
```

Comment:
Are you sure about this one?

Reply:
Yes Sir, the corrections are done according to the "Vale" suggestions.